packages/fxa-auth-server/lib/constants.js

```diff
@@ -0,0 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = {
+  SESSION_TOKEN_OAUTH_SCOPE: 'https://identity.mozilla.com/tokens/session',
```

Comment:
You could also consider lifting the "oldsync" scope into this file, it's currently repeated in a few places as a literal string.